### PR TITLE
add 7.3 in basic features

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

add 7.3 columns in basic features

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -20,227 +20,227 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| 扩展统计信息（多列） | E | E | E | E | E | E | E | E | E | E | N |
+| 统计反馈 | N | N | N | N | 已废弃 | 已废弃 | E | E | E | E | E |
+| [统计信息自动更新](/statistics.md#自动更新) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [快速分析](/system-variables.md#tidb_enable_fast_analyze) | E | E | E | E | E | E | E | E | E | E | E |
+| [动态裁剪](/partitioned-table.md#动态裁剪模式) | Y | Y | Y | Y | Y | E | E | E | E | N | N |
+| [收集部分列的统计信息](/statistics.md#收集部分列的统计信息) | E | E | E | E | E | E | N | N | N | N | N |
+| [限制统计信息的内存使用量](/statistics.md#统计信息收集的内存限制) | E | E | E | E | E | N | N | N | N | N | N |
+| [随机采样约 10000 行数据来快速构建统计信息](/system-variables.md#tidb_enable_fast_analyze) | E | E | E | E | E | E | E | E | E | E | E |
+| [锁定统计信息](/statistics.md#锁定统计信息) | E | E | E | E | N | N | N | N | N | N | N |
+| [轻量级统计信息初始化](/statistics.md#统计信息的加载) | Y | Y | E | N | N | N | N | N | N | N | N |
```
Comment (Oreoxmt) [inline basic-features.md:175]:
```suggestion
| [显示统计信息收集的进度](/sql-statements/sql-statement-show-analyze-status.md) | Y | N | N | N | N | N | N | N | N | N | N |

```

Comment (songrijie) [reply basic-features.md:175]: LGTM

File: basic-features.md [hunk 2]
```diff
@@ -20,227 +20,228 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [Stale Read](/stale-read.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | N | N |
+| [Follower Read](/follower-read.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [通过系统变量 `tidb_snapshot` 读取历史数据](/read-historical-data.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Optimizer hints](/optimizer-hints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [MPP 执行引擎](/explain-mpp.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [MPP 执行引擎 - compression exchange](/explain-mpp.md#mpp-version-和-exchange-数据压缩) | Y | Y | Y | N | N | N | N | N | N | N | N |
+| [TiFlash Pipeline 执行模型](/tiflash/tiflash-pipeline-model.md) | E | E | N | N | N | N | N | N | N | N | N |
+| [索引合并](/explain-index-merge.md) | Y | Y | Y | Y | Y | Y | E | E | E | E | E |
+| [基于 SQL 的数据放置规则](/placement-rules-in-sql.md) | Y | Y | Y | Y | Y | E | E | N | N | N | N |
+| [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner) | E | E | E | E | E | E | E | E | E | E | E |
```
Comment (ran-huang) [inline basic-features.md:108]:
```suggestion
| [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner) | E | E | E | E | E | E | E | E | E | E | E |
| [Runtime Filter](/runtime-filter.md) | Y | N | N | N | N | N | N | N | N | N | N |
```

Comment (ran-huang) [reply basic-features.md:108]: @songrijie PTAL

**Review by Oreoxmt — APPROVED** [review]

File: basic-features.md [hunk 3]
```diff
@@ -20,227 +20,228 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [动态裁剪](/partitioned-table.md#动态裁剪模式) | Y | Y | Y | Y | E | E | E | E | N | N |
-| [Range COLUMNS 分区](/partitioned-table.md#range-columns-分区) | Y | Y | Y | N | N | N | N | N | N | N |
-| [Range INTERVAL 分区](/partitioned-table.md#range-interval-分区) | Y | Y | E | N | N | N | N | N | N | N |
+| 分区 | 7.3 | 7.2 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [Range 分区](/partitioned-table.md#range-分区) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Hash 分区](/partitioned-table.md#hash-分区) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Key 分区](/partitioned-table.md#key-分区) | Y | Y | Y | N | N | N | N | N | N | N | N |
+| [List 分区](/partitioned-table.md#list-分区) | Y | Y | Y | Y | Y | E | E | E | E | E | N |
+| [List COLUMNS 分区](/partitioned-table.md#list-columns-分区) | Y | Y | Y | Y | Y | E | E | E | E | E | N |
```
Comment (qiancai) [inline basic-features.md:151]:
```suggestion
| [List COLUMNS 分区](/partitioned-table.md#list-columns-分区) | Y | Y | Y | Y | Y | E | E | E | E | E | N |
| [List 和 List COLUMNS 分区的默认分区](/partitioned-table.md#默认的-list-分区) | Y | N | N | N | N | N | N | N | N | N | N |
```

Comment (qiancai) [reply basic-features.md:151]: @joey-yez PTAL

File: basic-features.md (hunk 2 above)
Comment (songrijie) [reply basic-features.md:108]: LGTM

File: basic-features.md [hunk 4]
```diff
@@ -20,227 +20,230 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [执行计划管理 (SPM)](/sql-plan-management.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [根据历史执行计划创建绑定](/sql-plan-management.md#根据历史执行计划创建绑定) | Y | Y | Y | E | N | N | N | N | N | N | N |
+| [下推计算结果缓存 (Coprocessor Cache)](/coprocessor-cache.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | E |
+| [Stale Read](/stale-read.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | N | N |
+| [Follower Read](/follower-read.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [通过系统变量 `tidb_snapshot` 读取历史数据](/read-historical-data.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Optimizer hints](/optimizer-hints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [MPP 执行引擎](/explain-mpp.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [MPP 执行引擎 - compression exchange](/explain-mpp.md#mpp-version-和-exchange-数据压缩) | Y | Y | Y | N | N | N | N | N | N | N | N |
+| [TiFlash Pipeline 执行模型](/tiflash/tiflash-pipeline-model.md) | E | E | N | N | N | N | N | N | N | N | N |
```
Comment (joey-yez) [inline basic-features.md:105]:
```suggestion
| [TiFlash Pipeline 执行模型](/tiflash/tiflash-pipeline-model.md) | E | E | N | N | N | N | N | N | N | N | N |
| [TiFlash Repllica Read](/system-variables.md#tiflash_replica_read-从-v730-版本开始引入) | Y | N | N | N | N | N | N | N | N | N | N |
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**